oasst-shared/oasst_shared/model_configs.py

```diff
@@ -0,0 +1,83 @@
+import pydantic
+
+
+class ModelConfig(pydantic.BaseModel):
+    model_id: str
+    max_input_length: int = 512
+    max_total_length: int = 1024
+    quantized: bool = False
```

Comment:
Quantize currently means float16 always? We may have to update this is in the future in case we want to support int8/int4 ..

Reply:
Quantize means int8. it's a built-in of the HF server. by default our models are float16. int4 is going to be difficult...